client/src/main/com/sinch/sdk/domains/conversation/MessagesService.java

```diff
@@ -34,7 +43,23 @@ public interface MessagesService {
    *     com.sinch.sdk.domains.conversation.models.v1.messages.response.SendMessageResponse})
    * @since 1.0
    */
-  SendMessageResponse send(SendMessageRequest request);
```

Comment:
Yes, I did not yet identified utility.
So instead of having to, maintain a feature not yet identified as useful/required I did not retain it.
But I'm interesting if you already identified a feature and no problem to make it public and maintain it.

Reply:
Sure, my use case is the following: when handling the conversation event, I have a bot that reads the input message from the contact. The message that the chatbot will send back is delegated to another service that can produce a TextMessage, a LocationMessage... or anything else. Thus, I need a `send` method that takes any kind of message as a parameter to reply to the contact.

Reply:
Good point.
I think I'll need this kind of requirements too when it will time about webhook supports 
Thank you